src/main/java/seedu/storemando/ui/MainWindow.java

```diff
@@ -137,14 +138,15 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
     }
 
     /**
-     * Opens the help window or focuses on it if it's already opened.
+     * Opens storemando's user guide on user's browser if user's desktop supports it.
+     * Otherwise, opens the help window or focuses on it if it's already opened.
      */
     @FXML
     public void handleHelp() {
-        if (!helpWindow.isShowing()) {
-            helpWindow.show();
+        if (BrowserUtil.isBrowserAccessible() && BrowserUtil.isConnectedToInternet()) {
```

Comment:
Instead of calling 2 booleans, maybe inside BrowserUtil class create a boolean to check for these 2 booleans, since at this class we don't have to know the details. Maybe can create a boolean called canOpenBrowser :)

Reply:
Thanks for the suggestion. Fixed!